packages/dds/counter/src/counter.ts

```diff
@@ -196,11 +196,11 @@ export class SharedCounter extends SharedObject<ISharedCounterEvents> implements
     }
 
     /**
-     * Not implemented.
      *
```

Comment:
you should use an @inheritdoc tag here. Search codebase for "inheritdoc" for syntax (i always forget). that will remove the api report changes.

Reply:
Added the tag, but the api report still has the change

Reply:
mentioned this in chat, but so you can reference it in case anyone asked: some other members of our team have already filed an issue on API-extractor for this policy: https://github.com/microsoft/rushstack/issues/3462 . Thanks for calling this out.